report_thesis/src/sections/background.tex

```diff
@@ -316,7 +316,59 @@ \subsubsection{Extra Trees Regressor (ETR)}
 As a tradeoff, \gls{etr} is less interpretable than a single decision tree, as the added randomness can introduce more bias than \gls{rf}.
 However, it often achieves better generalization performance, especially in high-dimensional or noisy datasets.
 
-\subsubsection{Gradient Boosting Regressor (GBR)}
+\subsubsection{Gradient Boosting Regression (GBR)}\label{sec:gradientboost}
+In this section we introduce \gls{gbr} primarily based on \citet{James2023AnIS}.
+\gls{gbr} is an ensemble learning technique that uses a sequence of weak learners, typically decision trees, to build a strong predictive model.
+
+However, in order to explain \gls{gbr}, it is helpful to build on the concepts of ensemble learning and decision trees.
+Ensemble learning is a technique in machine learning where multiple models, known as \textit{weak learners}, are combined to produce more accurate predictions.
+Mathematically, ensemble learning can be defined as combining the predictions of $M$ weak learners to form a final prediction $\hat{y}$, such that:
+\begin{equation}
+    \hat{y} = \sum_{m=1}^{M} \alpha_m \hat{y}_m,
+\end{equation}
+where $\hat{y}_m$ is the prediction of the $m$-th weak learner and $\alpha_m$ is the weight assigned to the $m$-th weak learner.
+While there are various choices for weak learners, decision trees are a common choice.
+Decision trees are a core component of gradient boosting methods. 
+They partition the data into subsets based on feature values, aiming to create groups where data points have similar predicted outcomes. 
+The quality of a split is measured using various metrics. Some common metrics include impurity, information gain, and Gini index. 
```

Comment:
I would argue this line doesn't really bring any useful information - listing terms without explaining them does not provide value It might be better to instead mention, on line 335, that "once optimal splits are identified using some evaluation metric". 

What do you think?

Reply:
Then it would be something like:
"Once optimal splits are identified, using metrics such as impurity, information gain or Gini index, the tree is constructed by repeatedly partitioning the data until a stopping criterion is met."

Is this to your liking?

Reply:
Gini is impurity, no?
Called Gini impurity.

And missing entropy?

> Decision trees are fundamental to gradient boosting techniques. They work by recursively partitioning the dataset into smaller subsets based on feature values. Each split is chosen to maximize the homogeneity of the resulting subsets, where data points within each subset have similar predicted outcomes. The effectiveness of a split is measured using metrics such as information gain, Gini index, and entropy. These metrics help in selecting the most informative splits to enhance the model's predictive accuracy.

Current text is a little informal.

Also, we've already described decision trees for ETR. I vote we centralize the descriptions, as I mentioned in my earlier review comment. It'll get confusing otherwise.